### ISSUE TITLE
Hit Counter

### ISSUE THREAD

**Issue by Andrek25** [runelite/runelite#10992]
A hit counter to show a accurate % while killing a monster, something like this:

Right Click -> [Reset All] [Reset]
Demonic Gorilla
Melee: 485 / 645 Hits (75.19%)
Ranged: 495 / 586 Hits (84.47%)

Maybe add a Max Hit Done could be usefull too.

This is good to test gear and calculate dps (damage per second).

**Comment by Nightfirecat** [comment]
I'm not sure I understand what the numbers you gave mean. 485/645 hits dealt damage? Wouldn't that be misleading given that you can still hit 0 even if you attack with the "right" attack style, and even if you were to roll a hit? (but happen to roll 0 damage)
What is the ideal use case for this? Demonic Gorillas (as your request lists) and what else? This seems like it would be somewhat narrow.

**Comment by Andrek25** [comment]
485 successful hits of 645 hits done this gives us a 75.19% percent of accuracy.

I know that you can hit 0 even if your hit was successful but this method is still a good for calculating dps.

This can be useful for most monster, you can know an average of accuracy. I remember when i was killing Skeletal Wyverns using:

- Armadyl Helmet
- Ava's Accumulator
- Glory Amulet
- Broad Bolts
- Dragon Crossbow
- Blessed D'hide Body
- Twisted Buckler
- Blessed D'hide Chaps
- Blessed D'hide Bracers
- Blessed D'hide Boots
- Archers Ring (i).

And i bought a Necklace of Anguish then i was wondering if my accurate was incremented, i wish i could see how it affected to my DPS, something like 78.42% to 79.89%.

In short words: count each hit you make to a monster to get a percent of accuracy.

**Comment by Nightfirecat** [comment]
Why would this be better than using something like [Bitterkoekje's DPS calculator](https://www.reddit.com/r/2007scape/comments/64wtan/dps_calculator_by_bitterkoekje/)?

**Comment by Andrek25** [comment]
Some people don't know how to use [Bitterkoekje's DPS calculator](https://www.reddit.com/r/2007scape/comments/64wtan/dps_calculator_by_bitterkoekje/), and this is a true accurate.

I just think this is useful and **easy** to use.

**Comment by JacoboMoral** [comment]
> Some people don't know how to use [Bitterkoekje's DPS calculator](https://www.reddit.com/r/2007scape/comments/64wtan/dps_calculator_by_bitterkoekje/)

As far as I know, the official [OSRS Wiki](https://oldschool.runescape.wiki/) is working on a [DPS Calculator](https://www.reddit.com/r/2007scape/comments/fccxat/removal_of_weakness_parameter_from_wiki_monster/) which will probably be easier to use than Google Sheets:

> This will be complemented with the release of a DPS Calculator to truly figure out what is the best against each monster.

**Comment by cloakd** [comment]
> Some people don't know how to use [Bitterkoekje's DPS calculator](https://www.reddit.com/r/2007scape/comments/64wtan/dps_calculator_by_bitterkoekje/), and this is a 100% accurate.
> 
> I just think this is useful and **easy** to use.

I recently built the DPS Calculator spreadsheet into a web tool for everyone to use: http://osrs-genie.com/

This is a great feature suggestion, im currently working on implementing this calculator into runelite so will add the hit chance as an output. Tracking ticket #350
